tensilelite/Tensile/Source/lib/source/ContractionSolution.cpp

```diff
@@ -2964,21 +2973,14 @@ namespace TensileLite
                 auto numWGS = getNumWorkGroups(problem, sizeMapping);
                 size += problem.amaxd().elementBytes() * numWGS;
             }
-
```

Comment:
The actual change of this PR is removing this out-of-date code section, which was used for old design single kernel.